cognite/neat/core/rules/importer/owl2excel.py

```diff
@@ -0,0 +1,358 @@
+    """Parse owl metadata from graph to pandas dataframe.
+
+    Parameters
+    ----------
+    graph : Graph
+        Graph containing owl metadata
+    parsing_config : dict, optional
+        Configuration for parsing, by default PARSING_METADATA
+
+    Returns
+    -------
+    pd.DataFrame
+        Dataframe containing owl metadata
+    """
+    query = """
+    SELECT ?namespace ?prefix ?dataModelName ?cdfSpaceName ?version ?isCurrentVersion
+           ?created ?updated ?title ?description ?creator ?contributor ?rights ?license
+    WHERE {
+        ?namespace a owl:Ontology .
+        OPTIONAL {?namespace owl:versionInfo ?version }.
+        OPTIONAL {?namespace dcterms:creator ?creator }.
+        OPTIONAL {?namespace dcterms:title ?title }.
+        OPTIONAL {?namespace dcterms:contributor ?contributor }.
+        OPTIONAL {?namespace dcterms:modified ?updated }.
+        OPTIONAL {?namespace dcterms:created ?created }.
+        OPTIONAL {?namespace dcterms:description ?description }.
+        OPTIONAL {?namespace dcterms:rights|dc:rights ?rights }.
+        OPTIONAL {?namespace dcterms:license|dc:license ?license }.
+        FILTER (!isBlank(?namespace))
+        FILTER (!bound(?description) || LANG(?description) = "" || LANGMATCHES(LANG(?description), "en"))
+        FILTER (!bound(?title) || LANG(?title) = "" || LANGMATCHES(LANG(?title), "en"))
+    }
+    """
+
+    raw_df = pd.DataFrame(list(graph.query(query)), columns=parsing_config["header"])
+    if raw_df.empty:
+        return raw_df
+    raw_df.replace(np.nan, "", regex=True, inplace=True)
+
+    clean_list = [
+        [
+            raw_df.namespace.unique()[0],
+            raw_df.prefix.unique()[0],
+            raw_df.dataModelName.unique()[0],
+            raw_df.cdfSpaceName.unique()[0],
+            raw_df.version.unique()[0],
+            True,
+            raw_df.created.unique()[0],
+            raw_df.updated.unique()[0],
+            raw_df.title.unique()[0],
+            raw_df.description.unique()[0],
+            ", ".join(list(raw_df.creator.unique())),
+            ", ".join(list(raw_df.contributor.unique())),
+            raw_df.rights.unique()[0],
+            raw_df.license.unique()[0],
+        ]
+    ]
+
+    return pd.DataFrame(clean_list, columns=parsing_config["header"]).T
+
+
+# TODO: these are to be read from Class pydantic model
+PARSING_CONFIG_CLASSES = {
+    "helper_row": ("Data Model Definition", "", "", "", "State", "", "", "Knowledge acquisition log", "", "", ""),
+    "header": (
+        "Class",
+        "Name",
+        "Description",
+        "Parent Class",
+        "Deprecated",
+        "Deprecation Date",
+        "Replaced By",
+        "Source",
+        "Source Entity Name",
+        "Match",
+        "Comment",
+    ),
+}
+
+
+def _parse_owl_classes_df(graph: Graph, parsing_config: dict = PARSING_CONFIG_CLASSES) -> pd.DataFrame:
```

Comment:
Same here with the `Parsing_Config_Classes`.

Reply:
Done!